src/App.test.js

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
-import { mount, shallow } from 'enzyme';
+import { render, screen, fireEvent } from '@testing-library/react';
```

Comment:
Please note, that I switched Enzyme for RTL as David mentioned that this is what you use today.